assets/dev/js/editor/elements/views/container.js

```diff
@@ -147,8 +147,22 @@ const ContainerView = BaseElementView.extend( {
 					newIndex++;
 				}
 
+				// Prevent the user from dragging a parent container into a child container
+				const draggedId = draggedView.getContainer().id;
+
+				let draggingParent = false,
```

Comment:
move this line out of the new `if` block

Reply:
I don't think that is necessary, but you can double check that for me.